vyper/semantics/analysis/imports.py

```diff
@@ -147,10 +148,11 @@ def _handle_ImportFrom(self, node: vy_ast.ImportFrom):
     def _add_import(
         self, node: vy_ast.VyperNode, level: int, qualified_module_name: str, alias: str
     ) -> None:
-        compiler_input, ast = self._load_import(node, level, qualified_module_name, alias)
-        node._metadata["import_info"] = ImportInfo(
-            alias, qualified_module_name, compiler_input, ast
-        )
+        with tag_exceptions(node):
```

Comment:
let's move this to the highest scope where we have access to node -- i think inside the loop body on line 108